src/pages/articles/software-engineering/singleton/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Singleton"
+parent: "to-be-sorted"
```

Comment:
Meta will be created automatically, at the minute, all of this will be overwritten except for the title